query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryServiceConfig.java

```diff
@@ -62,12 +62,36 @@ private RequestHandlerConfig(Config config) {
   public static class RequestHandlerClientConfig {
     private static final String CONFIG_PATH_TYPE = "type";
     private static final String CONFIG_PATH_CONNECTION_STRING = "connectionString";
+    private static final String CONFIG_PATH_HOST = "host";
+    private static final String CONFIG_PATH_PORT = "port";
+    private static final String CONFIG_PATH_DATABASE_NAME = "databaseName";
+    private static final String CONFIG_PATH_USER = "user";
+    private static final String CONFIG_PATH_PASSWORD = "password";
     String type;
     String connectionString;
+    String host;
+    int port;
+    String databaseName;
+    String user;
+    String password;
 
     private RequestHandlerClientConfig(Config config) {
       this.type = config.getString(CONFIG_PATH_TYPE);
-      this.connectionString = config.getString(CONFIG_PATH_CONNECTION_STRING);
+      if (config.hasPath(CONFIG_PATH_CONNECTION_STRING)) {
```

Comment:
Let's simplify this - connection string + user + password, so no if, those are must config.

Reply:
Done